cfme/utils/template/base.py

```diff
@@ -98,17 +99,27 @@ def stream_url(self):
         """
         urls = cfme_data.basic_info.cfme_images_url
 
-        if self._stream_url:
-            return self._stream_url
-        elif self.stream in urls:
+        if self.stream in urls:
             return urls[self.stream]
         else:
             logger.error("Stream %s is not listed in cfme_data. "
-                         "Please specify stream URL with --stream_url", self.stream)
+                         "Please specify stream with --stream", self.stream)
             raise TemplateUploadException("Cannot get stream URL.")
 
     @property
-    def image_url(self):
+    def stream_short(self):
+        """Short name for the stream, like '59z' or 'gaprindashvili'"""
+        if 'downstream-' in self.stream:
+            # makes downstream-nightly into the 'nightly' short stream (folder names)
+            return self.stream.replace('downstream-', '')
+        elif 'upstream_' in self.stream:
+            return self.stream.replace('upstream_', '')
```

Comment:
Q: afaik there are different upstream_nightly streams and etc. should we handle them here as well ?
btw, it would be good to make some conversion function in miq_version to convert short stream to long one and back

Reply:
It looks like I ended up not actually using this method? Wrote it for dealing with sprout shortened stream names, but I think the miq_version changes ended up making it unnecessary.